kbase/execution_engine2
# DATAUP-536 Bad jobs reaper

# Description of PR purpose/changes

* Adds a simple job reaper.
* Fixes some pymongo connection string issues
* Another time we can try with the APS scheduler (some work at https://github.com/kbase/execution_engine2/tree/aps_scheduler)

# Jira Ticket / Github Issue #

# Testing Instructions
* You can see jobs being changed in #ee_notifications channel
* You can run the script manually from the ee2 container
* Deployed in CI

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/data-upload-project/development
- [x] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the Github Actions build passes)

# Updating Version and Release Notes (if applicable)
